programs/clearing_house/src/state/oracle.rs

```diff
@@ -9,6 +9,85 @@ use std::cmp::max;
 use switchboard_v2::decimal::SwitchboardDecimal;
 use switchboard_v2::AggregatorAccountData;
 
+#[derive(Default, AnchorSerialize, AnchorDeserialize, Clone, Copy, Eq, PartialEq, Debug)]
+pub struct HistoricalOracleData {
+    // use u64?
+    pub last_oracle_price: i128,
+    pub last_oracle_conf: u128,
+    pub last_oracle_delay: i64,
+    pub last_oracle_price_twap: i128,
+    pub last_oracle_price_twap_5min: i128,
+    pub last_oracle_price_twap_ts: i64,
```

Comment:
would we also want to store last_oracle_validity? not sure just throwing it out there

Reply:
think as a follow up branch we can consider